swank.asd

```diff
@@ -19,19 +19,87 @@
+                  (:file "swank")
+                  ,@(mapcar (lambda (component)
+                              `(:file ,component :pathname ,(concatenate 'string "swank/" component)))
+                            impl-files)
+                  (:file "asdf-init")
+                  (:module "contrib"
+                   :components ((:no-load-file "swank-c-p-c")
+                                (:no-load-file "swank-arglists"
+                                 :depends-on ("swank-c-p-c"))
+                                (:no-load-file "swank-asdf")
+                                (:no-load-file "swank-clipboard")
+                                (:no-load-file "swank-fancy-inspector")
+                                (:no-load-file "swank-fuzzy"
+                                 :depends-on ("swank-c-p-c"))
+                                (:no-load-file "swank-hyperdoc")
+                                (:no-load-file "swank-indentation")
+                                (:no-load-file "swank-repl")
+                                (:no-load-file "swank-macrostep")
+                                (:no-load-file "swank-mrepl")
+                                ;; (:no-load-file "swank-listener-hooks"
+                                ;;  :depends-on ("swank-repl"))
+                                (:no-load-file "swank-media")
+                                (:no-load-file "swank-package-fu")
+                                (:no-load-file "swank-presentations"
+                                 :depends-on ("swank-repl"))
+                                (:no-load-file "swank-quicklisp")
+                                ;; (:no-load-file "swank-presentation-streams"
+                                ;;  :depends-on ("swank-repl" "swank-presentations"))
+                                (:no-load-file "swank-sbcl-exts"
+                                 :depends-on ("swank-arglists"))
+                                (:no-load-file "swank-snapshot"
+                                 :depends-on ("swank-repl"))
+                                (:no-load-file "swank-sprof"))))
+     :depends-on (#+sbcl #:sb-bsd-sockets)
+     :perform (load-op :after (op swank)
+                (load-user-init-file))))
+
+#+(or cmu scl sbcl openmcl clozurecl lispworks allegro clisp armedbear cormanlisp ecl)
+(define-swank-system
+  #+cmu (("source-path-parser" "source-file-cache") ("cmucl" "gray"))
+  #+scl (("source-path-parser" "source-file-cache") ("scl" "gray"))
+  #+sbcl (("source-path-parser" "source-file-cache") ("sbcl" "gray"))
+  #+(or openmcl clozurecl) (("metering" ) ("ccl" "gray"))
+  #+lispworks (() ("lispworks" "gray"))
+  #+allegro (() ("allegro" "gray"))
+  #+clisp (("xref" "metering") ("clisp" "gray"))
+  #+armedbear (() ("abcl"))
+  #+cormanlisp (() ("corman" "gray"))
+  #+ecl (("source-path-parser" "source-file-cache") ("ecl" "gray")))
+
+#-(or cmu scl sbcl openmcl clozurecl lispworks allegro clisp armedbear cormanlisp ecl)
```

Comment:
I recommend alphabetic sorting of #+ clauses.

Reply:
Done.